### ISSUE TITLE
Data duplicate onLoad 

### ISSUE THREAD

**Issue by dev-badace** [partykit/partykit-text-editor-example#3]
see this [discord issue thread](https://discord.com/channels/1051830863576453180/1175147713692708895)
tldr :- this demo is fetching data from supabase client onLoad, and using it to create a new Y.Doc , however that data is not Y.encodeStateAsUpdate so everytime it recreates the entire doc with a new clientId, and the rest of the clients then sync the doc with their data, which then leads to duplicate doc

there are 2 potential solution to this, 
1. either store the  `Y.encodeStateAsUpdate` in a new field in supabase and use it to initialize the new doc `onLoad`
2. only use supabase client to update/sync the database, and store the doc using party.storage

both of these solutions are shown in the [discord thread](https://discord.com/channels/1051830863576453180/1175147713692708895)

**Comment by threepointone** [comment]
I've updated the example to take this approach. Sorry for the delay, and thank you!
